splinterd/sample_configs/splinterd.toml.example

```diff
@@ -128,3 +128,32 @@ oauth_client_secret = "<client-secret>"
 
 # Redirect URL for the OAuth provider used by the REST API
 oauth_redirect_url = "http://localhost:8080/oauth/callback"
+
```

Comment:
Defaults should be made obvious, similar to the beginning of the file. Separately, we should fix the oauth lines of the file that don't make the defaults obvious.